### ISSUE TITLE
Autocomplete in Data Source Configuration Dialog

### ISSUE THREAD

**Issue by joemoe** [raintank/crate-datasource#6]
It would be super cool to have autocomplete of schema, table and time column in the Data Source Configuration Dialog.

**Comment by alexanderzobnin** [comment]
It may be a bit hard. But I can try to implement it.

**Comment by jodok** [comment]
i agree, this is lower prio / cosmetics. let's keep the issue here right now and tackle it after we finished the rest of the more important issues.

**Comment by alexanderzobnin** [comment]
Yes, datasource setup is a single-time action, so I think, it's not a big problem right now.
